README.md

```diff
@@ -54,3 +54,8 @@ To run the tools that start the local development server:
 ```bash
 make
 ```
+
+To install pre-commit to help with pull requests
+```bash
+pre-commit install
+```
```

Comment:
Instead of including this here, let's move this to be part of the bootstrap script. By doing that, anyone who wants to work on the repo will have the proper setup without running something extra.